packages.yml

```diff
@@ -1,3 +1,6 @@
 packages:
-  - package: fivetran/quickbooks_source
-    version: 0.1.2
\ No newline at end of file
+  # - package: fivetran/quickbooks_source
+  #   version: [">=0.2.0","<0.3.0"]
+  - git: https://github.com/fivetran/dbt_quickbooks_source.git
```

Comment:
This will be removed once [dbt_quickbooks_source PR #9](https://github.com/fivetran/dbt_quickbooks_source/pull/9) is approve.